orchestration/tests/test_prompt_file_lifecycle.py

```
@@ -0,0 +1,452 @@
+Prompt File Lifecycle Tests
+
+Tests the complete lifecycle of task prompt files to ensure:
+1. Files are created with correct content for each task
+2. Files are cleaned up appropriately based on age and context
+3. No cross-contamination between different orchestration runs
+4. Memory usage doesn't grow unbounded from accumulated files
+
+This addresses the specific lifecycle management gap that caused the stale task bug.
+"""
+
+import glob
```

Ruff (0.12.2)

`glob` imported but unused

Remove unused import: `glob`

(F401)